core/update_translations.sh

```diff
@@ -12,11 +12,12 @@ mv "$translations/es.properties" "$i18n/displayStrings_es.properties"
 mv "$translations/ja.properties" "$i18n/displayStrings_ja.properties"
 mv "$translations/pt.properties" "$i18n/displayStrings_pt.properties"
 mv "$translations/ru.properties" "$i18n/displayStrings_ru.properties"
-mv "$translations/zh_CN.properties" "$i18n/displayStrings_zh.properties"
+mv "$translations/zh_CN.properties" "$i18n/displayStrings_zh-hans.properties"
+mv "$translations/zh_TW.properties" "$i18n/displayStrings_zh-hant.properties"
```

Comment:
To support that you need to update also `tx pull -l de,el_GR,es,ja,pt,ru,zh_CN,vi,th_TH,fa,fr,pt_BR` to `tx pull -l de,el_GR,es,ja,pt,ru,zh_CN,zh_TW,vi,th_TH,fa,fr,pt_BR`